CalcualtorLogic.js

```diff
@@ -91,6 +91,7 @@ class Calculator {
         const clearingButton = document.createElement("button");
         clearingButton.classList.add("button");
         const clearingButtonText = document.createTextNode("Clear");
+        clearingButton.id = "clearingButton"
```

Comment:
you don't need `id` 
your clear button is fine, just need to update the UI and fix the `bound` issue